GoTeamApp/GoTeamApp/LabelTableViewCell.swift

```diff
@@ -1,14 +1,61 @@
 import UIKit
 
+protocol LabelTableViewCellDelegate {
+    func editButtonAction(cell: LabelTableViewCell)
+    func filterTasksForSelectedLabelAction(cell: LabelTableViewCell)
+    func deleteLabelCell(sender : LabelTableViewCell)
+}
+
 class LabelTableViewCell: UITableViewCell {
 
     @IBOutlet weak var labelName: UILabel!
+    @IBOutlet weak var editLabelButton: UIButton!
+    @IBOutlet weak var tasksFilterButton: UIButton!
+    
+    
+    @IBOutlet weak var deleteView: UIView!
+    @IBOutlet weak var topView: UIView!
+    var delegate: LabelTableViewCellDelegate?
```

Comment:
FYI: This needs to be weak var, otherwise it will lead to a circular reference. Have a look at the other UITableViewCells when you get the chance. I can merge for now, since it doesn't break functionality, but it can lead to memory leaks.